Gems/ROS2/docs/guides/ros2-gem.md

```diff
@@ -20,6 +20,7 @@ You can browse Doxygen-generated documentation on [Gem's GitHub page](https://ro
   - ROS2GNSSSensorComponent
   - ROS2IMUSensorComponent
   - ROS2LidarSensorComponent
+  - ROS2Lidar2DSensorComponent
```

Comment:
This is now in the documentation (the user guide here is probably going to be removed). Adding a component page to documentation can be done later.

Reply:
It is not clear what I could improve here, for now.